tests/modules/nf-core/blast/blastp/main.nf

```diff
@@ -10,5 +10,33 @@ workflow test_blast_blastp {
     input = [ file(params.test_data['sarscov2']['genome']['proteome_fasta'], checkIfExists: true) ]
 
     BLAST_MAKEBLASTDB ( input )
-    BLAST_BLASTP ( [ [id:'test'], input ], BLAST_MAKEBLASTDB.out.db )
+    out_ext = '' // empty test case to check default
+    BLAST_BLASTP ( [ [id:'test'], input ], BLAST_MAKEBLASTDB.out.db, out_ext )
+}
+
+workflow test_blast_blastp_xml {
+
+    input = [ file(params.test_data['sarscov2']['genome']['proteome_fasta'], checkIfExists: true) ]
+
+    BLAST_MAKEBLASTDB ( input )
+    out_ext = 'xml'
+    BLAST_BLASTP ( [ [id:'test'], input ], BLAST_MAKEBLASTDB.out.db, out_ext )
+}
+
+workflow test_blast_blastp_tsv {
+
+    input = [ file(params.test_data['sarscov2']['genome']['proteome_fasta'], checkIfExists: true) ]
+
+    BLAST_MAKEBLASTDB ( input )
+    out_ext = 'tsv'
+    BLAST_BLASTP ( [ [id:'test'], input ], BLAST_MAKEBLASTDB.out.db, out_ext )
+}
+
+workflow test_blast_blastp_csv {
+
+    input = [ file(params.test_data['sarscov2']['genome']['proteome_fasta'], checkIfExists: true) ]
+
+    BLAST_MAKEBLASTDB ( input )
+    out_ext = 'csv'
+    BLAST_BLASTP ( [ [id:'test'], input ], BLAST_MAKEBLASTDB.out.db, out_ext )
```

Comment:
Why are we adding anything to the pytest-test workflow tests? Not sure if this is duplicating the logic in nf-test, if so no problem. Mainly don't want to be missing anything in nf-test, because we'll delete these files soon.

Reply:
Aye, nf-tests are a superset of pytests. I was just testing around a bit with pytest as well.